src/scene/vcVerticalMeasureTool.cpp

```diff
@@ -183,6 +198,18 @@ void vcVerticalMeasureTool::HandleImGui(vcState *pProgramState, size_t *pItemID)
   }
 }
 
+void vcVerticalMeasureTool::ResetLabelText()
+{
+  for (auto &label : m_labelList)
+  {
+    if (label.pText)
+    {
+      udFree(label.pText);
+      label.pText = nullptr;
+    }
```

Comment:
All of this is done by `udFree`, it checks if the provided pointer is null to avoid crashes, it also sets the pointer to null before it frees the pointer. So you can collapse this down to just `udFree(label.pText);`.

Reply:
Fixed.